### PR TITLE
feat: add semantic release

### PR DESCRIPTION
**Pull request by joshje**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +21,14 @@
...
-    "eslint": "^5.3.0",
     "minimist": "^1.2.0",
     "moment": "^2.22.2",
     "request": "^2.87.0",
     "request-promise-native": "^1.0.5",
     "uuid-validate": "0.0.2"
+  },
+  "devDependencies": {
+    "eslint": "^5.3.0",
+    "semantic-release": "^15.9.8"
```
Comment (darscan) [inline package.json]: I'd remove this - we're using `npx` above anyway

Comment (joshje) [reply package.json]: How will the module be available if it isn't installed? Or maybe install it as part of the `travis.yml`?

Comment (darscan) [reply package.json]: > How will the module be available if it isn't installed?

That's sorta the point of `npx` - running binaries from modules that are not necessarily installed into the project. https://www.npmjs.com/package/npx#description :)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+sudo: false
+dist: trusty
+language: node_js
+notifications:
+  email: false
+  slack: false
+node_js:
+  - "8"
+after_success:
+  npm run semantic-release
```
Comment (darscan) [inline .travis.yml]: lol, noooo.. you've gone the other way!

Comment (darscan) [reply .travis.yml]: Let's use `npx` here, and drop it from our deps

Comment (joshje) [reply .travis.yml]: 🤣

**Pull request merged**